omnibus-base-mainnet-andromeda.toml

```
@@ -1,5 +1,5 @@
 name = "synthetix-omnibus"
-version = "20"
+version = "21"
```

Same here, no need to bump version. For cannon this update is noop
```suggestion
version = "20"
```